Description: Remove crappy upstream Debian packaging stuff
 This is cherry-picked from 3e8413be302d8c75da8996a6e3b73d619b2e7b34
Author: Julien Danjou <acid@debian.org>

---
Origin: upstream
Forwarded: not-needed
Last-Update: 2011-11-25

--- horizon-2012.1~e1.orig/horizon/Makefile
+++ horizon-2012.1~e1/horizon/Makefile
@@ -1,6 +1,5 @@
 PYTHON=`which python`
 DESTDIR=/
-BUILDIR=$(CURDIR)/debian/horizon
 PROJECT=horizon
 
 all:
@@ -9,7 +8,6 @@ all:
 	@echo "make source - Create source package"
 	@echo "make install - Install on local system"
 	@echo "make buildrpm - Generate a rpm package"
-	@echo "make builddeb - Generate a deb package"
 	@echo "make clean - Get rid of scratch and byte files"
 
 buildout: ./bin/buildout
@@ -27,17 +25,7 @@ install:
 buildrpm:
 	$(PYTHON) setup.py bdist_rpm --post-install=rpm/postinstall --pre-uninstall=rpm/preuninstall
 
-builddeb:
-	# build the source package in the parent directory
-	# then rename it to project_version.orig.tar.gz
-	$(PYTHON) setup.py sdist $(COMPILE) --dist-dir=../
-	rename -f 's/$(PROJECT)-(.*)\.tar\.gz/$(PROJECT)_$$1\.orig\.tar\.gz/' ../*
-	# build the package
-	#dpkg-buildpackage -i -I -rfakeroot
-	dpkg-buildpackage -b -rfakeroot -tc -uc -D
-
 clean:
 	$(PYTHON) setup.py clean
-	$(MAKE) -f $(CURDIR)/debian/rules clean
 	rm -rf build/ MANIFEST
 	find . -name '*.pyc' -delete
--- horizon-2012.1~e1.orig/openstack-dashboard/Makefile
+++ horizon-2012.1~e1/openstack-dashboard/Makefile
@@ -1,6 +1,5 @@
 PYTHON=`which python`
 DESTDIR=/
-BUILDIR=$(CURDIR)/debian/openstack-dashboard
 PROJECT=openstack-dashboard
 
 all:
@@ -9,7 +8,6 @@ all:
 	@echo "make source - Create source package"
 	@echo "make install - Install on local system"
 	@echo "make buildrpm - Generate a rpm package"
-	@echo "make builddeb - Generate a deb package"
 	@echo "make clean - Get rid of scratch and byte files"
 
 buildout: ./bin/buildout
@@ -27,15 +25,6 @@ install:
 buildrpm:
 	$(PYTHON) setup.py bdist_rpm --post-install=rpm/postinstall --pre-uninstall=rpm/preuninstall
 
-builddeb:
-	# build the source package in the parent directory
-	# then rename it to project_version.orig.tar.gz
-	$(PYTHON) setup.py sdist $(COMPILE) --dist-dir=../
-	rename -f 's/$(PROJECT)-(.*)\.tar\.gz/$(PROJECT)_$$1\.orig\.tar\.gz/' ../*
-	# build the package
-	#dpkg-buildpackage -i -I -rfakeroot
-	dpkg-buildpackage -b -rfakeroot -tc -uc -D
-
 clean:
 	$(PYTHON) setup.py clean
 	$(MAKE) -f $(CURDIR)/debian/rules clean
--- horizon-2012.1~e1.orig/openstack-dashboard/setup.py
+++ horizon-2012.1~e1/openstack-dashboard/setup.py
@@ -26,13 +26,6 @@ def read(fname):
     return open(os.path.join(os.path.dirname(__file__), fname)).read()
 
 
-dst = 'debian/openstack-dashboard/var/lib/dash'
-os.system('rm -rf %s' % dst)
-shutil.copytree('tools', '%s/tools' % dst)
-shutil.copytree('dashboard', '%s/dashboard' % dst)
-shutil.copytree('local', '%s/local' % dst)
-
-
 setuptools.setup(
     name = 'openstack-dashboard',
     version = '0.4',
